src/simulate/main.py

```diff
@@ -42,31 +46,35 @@ def main():
 
     files = os.listdir(json_files_dir)
     logging.info('list of json files in the directory %s', files)
-    for file in files:
-        path_to_file = json_files_dir + "/" + file
-        try:
-            caliper_event = open(path_to_file, 'rb')
-        except IOError as e:
-            logging.error('cannot read the file %s due to %s', path_to_file, e)
-            continue
-        with caliper_event:
-            event = caliper_event.read()
+    for i in range(run_count):
+        for file in files:
+            path_to_file = json_files_dir + "/" + file
             try:
-                jsonEvent = json.loads(event)
-            except JSONDecodeError as e:
-                logging.error('Failed to Deserialize the caliper event %s ', e)
+                caliper_event = open(path_to_file, 'rb')
+            except IOError as e:
+                logging.error('cannot read the file %s due to %s', path_to_file, e)
                 continue
-            # make needed changes to the json events
-            event_transformer = Transformer(jsonEvent, config_yml_obj)
-            json_event_transformed = event_transformer.event_transformer()
+            with caliper_event:
+                event = caliper_event.read()
+                try:
+                    jsonEvent = json.loads(event)
+                except JSONDecodeError as e:
+                    logging.error('Failed to Deserialize the caliper event %s', e)
+                    continue
+                # make needed changes to the json events
+                event_transformer = Transformer(jsonEvent, config_yml_obj)
+                json_event_transformed = event_transformer.event_transformer()
```

Comment:
Since the transformer is called as `event_transformer.event_transformer()`, you should rename the method to `transform()`.  Then you could call `event_transformer.transform()`.

Reply:
Done